### PR TITLE
Proofread k8s operator guide

### PR DESCRIPTION
**Pull request by NickVolynkin**
Proofread the k8s operator guide

* Fix a few typos
* Add/change some articles
* Use English-language links for English version of the document
* Use the correct preposition: "on Kubernetes", not "in"
* Use a more informative title for better SEO
* Move a paragraph into the "Note" block above it

Fix code examples:

* Fix a couple of typos
* Use code-block Sphinx directive to enable the "Copy" button
* Add command line prompt to distinguish the command from output
* Hide the ever-changing hashes of tarantool-operator pods
* Update the instructions on delivering the tarantool-operator
  image into a Docker repository in a local network
  for image version 0.0.8.
* Fix inline code formatting

Fix a broken heading in Readme

### CODE REVIEW

File: doc/cartridge_kubernetes_guide/index.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ In the ``test-app`` directory, we get the app created from a template:
    ---
    ...
 
-   instances.yaml
+   instances.yml
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:99]: Cartridge CLI created files with `yml` extensions for me.

File: doc/cartridge_kubernetes_guide/index.rst [hunk 2]
```diff
@@ -313,52 +313,52 @@ joining a node or creating a replica set. The operator knows how to do
...
 a template. Run it with the default settings and get our example application
-that have 3 instances. If you define your own settings, you can deploy any
+that has 3 instances. If you define your own settings, you can deploy any
 application of any topology using the Tarantool Cartridge.
 
 Let's have a look at the settings in the ``values.yaml`` file.
 Comments provide a description of each parameter:
 
-.. code:: yaml
+.. code-block:: yaml
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:345]: `code-block` has the "Copy" button, just `code` doesn't.

File: doc/cartridge_kubernetes_guide/index.rst [hunk 3]
```diff
@@ -382,11 +382,11 @@ Comments provide a description of each parameter:
 
 With this configuration we will get the following:
 
-1. Tarantool Cartridge cluster called ``test-app``.
-2. Two replica sets in the cluster: ``routers`` and ``storages``.
-3. One tarantool instance in the ``routers`` replica set.
-4. Two instances, master and replica, in the ``storages`` replica set.
-5. Each replica set performs the roles listed in the ``RolesToAssign`` parameter.
+*  A Tarantool Cartridge cluster called ``test-app``.
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:385]: List items here just describe a set of objects, they aren't steps of an instruction. Hence, we don't need numbers.

File: doc/cartridge_kubernetes_guide/index.rst [hunk 4]
```diff
@@ -1,18 +1,18 @@
 .. _cartridge_kubernetes_guide:
 
 ================================================================================
-Kubernetes guide
+Tarantool Cartridge on Kubernetes
 ================================================================================
 
 This guide covers the full life cycle of a Tarantool Cartridge app--from
-developing the app to operating it in Kubernetes.
+developing the app to operating it on Kubernetes.
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:8]: The official Kubernetes documentaton uses "on", not "in".

File: doc/cartridge_kubernetes_guide/index.rst [hunk 5]
```diff
@@ -578,11 +580,11 @@ Update the ``values.yaml`` configuration file by specifying a new ``image.tag``:
      tag: "0.1.0-1-g4577716"
      pullPolicy: "IfNotPresent"
 
-Update the app in Kubernetes:
+Update the app on Kubernetes:
 
 .. code-block:: console
 
-   helm upgrade -f values.yaml test-app tarantool/cartridge --namespace tarantool
+   $ helm upgrade -f values.yaml test-app tarantool/cartridge --namespace tarantool
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:587]: without `$` the line can't be copied with a button

File: doc/cartridge_kubernetes_guide/index.rst [hunk 6]
```diff
@@ -666,17 +668,17 @@ To remove a cluster, execute the following command:
...
 
 After a while, all the pods of our application will disappear. Among the
-pods in the ``tarantool`` namespace, only Tarantool Kubernetes operator will
+pods in the ``tarantool`` namespace, only the Tarantool Kubernetes operator will
 remain.
 
 .. code-block:: console
 
-   $ kubectl -n tarantool get pods
+   $ kubectl get pods -n tarantool
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:676]: just for consistency with other uses of the same command

File: doc/cartridge_kubernetes_guide/index.rst [hunk 7]
```diff
@@ -779,21 +781,21 @@ Customization
...
-by making your own changes. Or use ``deployment.yaml`` and ``kubectl`` instead
+by making your own changes. You can also ``deployment.yaml`` and ``kubectl`` instead
 of ``helm``.
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Sidecar containers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-What it is? In Kubernetes, it is possible to create several
+What are they? With Kubernetes, it is possible to create several
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:791]: The heading is plural: containers.

File: doc/cartridge_kubernetes_guide/index.rst [hunk 8]
```diff
@@ -872,60 +874,60 @@ You can download the charts from the following links:
...
 And pack it into the archive:
 
 .. code-block:: console
 
-   $ docker save tarantool/tarantool-operator:0.0.6 | gzip > tarantool-operator-0.0.6.tar.gz
+   $ docker save tarantool/tarantool-operator:0.0.8 | gzip > tarantool-operator-0.0.8.tar.gz
 
 After delivering the archive with the container to the target location,
-you need to upload the image to your docker:
+you need to load the image to your Docker:
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:900]: It's the local Docker app, not some registry, so just "load".

File: doc/cartridge_kubernetes_guide/index.rst [hunk 9]
```diff
@@ -1,18 +1,18 @@
 .. _cartridge_kubernetes_guide:
 
 ================================================================================
-Kubernetes guide
+Tarantool Cartridge on Kubernetes
```
Comment (NickVolynkin) [inline doc/cartridge_kubernetes_guide/index.rst:4]: This is what people will search for on Google.

File: doc/cartridge_kubernetes_guide/index.rst (hunk 2 above)
Comment (vanyarock01) [reply doc/cartridge_kubernetes_guide/index.rst:345]: The "copy" button does not appear. It may not be supported with "yaml" format.

**Review by vanyarock01 — APPROVED** [review]

**Pull request merged**